tests/dir.props

```diff
@@ -47,8 +47,40 @@
     <CoreClrPackageVersion>1.0.4-beta-24325-02</CoreClrPackageVersion>
 
     <CoreFxVersionsIdentityRegex>^(?i)((System\..*)|(Microsoft\.CSharp)|(Microsoft\.NETCore.*)|(Microsoft\.Win32\..*)|(Microsoft\.VisualBasic))(?&lt;!TestData)$</CoreFxVersionsIdentityRegex>
+
+    <BaseDotNetBuildInfoUrl>https://raw.githubusercontent.com/dotnet/versions/master/build-info/dotnet/</BaseDotNetBuildInfoUrl>
+    <DependencyBranch>master</DependencyBranch>
```

Comment:
Would this need to be updated when the change is moved to release branch in future? If so, can we set it only if not already set, enabling it to be set via build system too?